src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/PerformanceController.php

```diff
@@ -66,6 +66,19 @@ public function indexAction(FormInterface $form = null)
             'servers' => $this->get('prestashop.adapter.memcache_server.manager')->getServers(),
         );
 
+        if (!in_array(
```

Comment:
this issue will concern a lot of migrated pages :/

Reply:
This is the list of controllers that need to be updated:
* [Addons](https://github.com/PrestaShop/PrestaShop/blob/1.7.3.x/src/PrestaShopBundle/Controller/Admin/Improve/AddonsStoreController.php)
* [Translations](https://github.com/PrestaShop/PrestaShop/blob/1.7.3.x/src/PrestaShopBundle/Controller/Admin/TranslationsController.php)

We have a big issue with Stock page /c @eternoendless, we don't have action to setup authorizations :/